src/runtime/config/configuration-qemu.toml.in

```diff
@@ -76,6 +76,13 @@ firmware_volume = "@FIRMWAREVOLUMEPATH@"
 # For example, `machine_accelerators = "nosmm,nosmbus,nosata,nopit,static-prt,nofw"`
 machine_accelerators="@MACHINEACCELERATORS@"
 
+# Qemu seccomp sandbox feature
+# comma-separated list of seccomp sandbox features to control the syscall access.
+# For example, `seccompsandbox= "on,obsolete=deny,elevateprivileges=deny,spawn=deny,resourcecontrol=deny"`
+# Please note, enable this feature may lead some performance cost, you may enable
+# /proc/sys/net/core/bpf_jit_enable to reduce the impact.
```

Comment:
If we have this suggestion here (perhaps a better place would be in docs), I think it would make sense to more clearly state how to enable this. Similarly, we may want to detail what performance impact this has (ie, on IO, on all syscalls, etc).

Reply:
@egernst I need to collect more data on the performance impact.